### ISSUE TITLE
Sharing to Instagram Stories

### ISSUE THREAD

**Issue by schwa23** [sindresorhus/Actions#256]
### Description

Instagram has a fairly simple sharing interface that uses a custom url scheme and a special pasteboard type. Sharing directly to instagram via shortcuts would entail placing content on the pasteboard as `com.instagram.sharedSticker.backgroundImage` and opening a URL `instagram-stories://share?source_application=%@`

[Documentation is here](https://developers.facebook.com/docs/instagram-platform/sharing-to-stories/)

**Comment by schwa23** [comment]
(it seems like the named clipboard function might suffice so feel free to close if this is too specific!)

**Comment by sindresorhus** [comment]
From the linked page:

> Beginning in January 2023, you must provide a Facebook AppID to share content to Instagram Stories. For more information, see Introducing an important update to Instagram Sharing to Stories. If you don't provide an AppID, your users see the error message "The app you shared from doesn't currently support sharing to Stories" when they attempt to share their content to Instagram.

**Comment by sindresorhus** [comment]
Not sure I will bother to apply for an AppID. However, what I could do is to make an action that allows setting custom pasteboard types.

**Comment by jdickens23** [comment]
Yeah that would work too! FWIW, I was able to put an arbitrary app id and it still worked so not sure how stringently they check that. But a generic 'advanced clipboard' action would be great :)

**Comment by sindresorhus** [comment]
I started implementing this, but it turns out only things in the foreground can put something on the clipboard on iOS, which means a shortcut action cannot unless it opens the Actions app, which is sub-optimal. However, I discovered a better solution. You can use the `Set Uniform Type Identifier` action to set the desired type on the content and then pass it to the built-in `Copy to Clipboard` action. Can you try that out? You need the latest TestFlight build for that: https://testflight.apple.com/join/fJGUrsZx
